throw语句  
 
运用在方法内部,抛出一个具体的异常对象  
	throw new 异常类("异常信息");终止方法
return 是返回一个值               throw 是返回一个错误 给调用者 (自带结束方法特性) 谁调用谁处理
然后调用者  用try{}catch(){}  来处理掉



throws语句

运用在方法声明之上   用于表示 当前方法不处理异常(处理不了),而是提醒该方法的调用者来处理异常(抛出异常)
一般是声明在方法签名右边
